app/src/main/java/com/keylesspalace/tusky/MainActivity.kt

```diff
@@ -350,6 +366,14 @@ class MainActivity : BottomSheetActivity(), ActionButtonActivity, HasAndroidInje
         }
     }
 
+    override fun onPrepareMenu(menu: Menu) {
+        super.onPrepareMenu(menu)
+
+        // If the main toolbar is hidden then there's no space in the top/bottomNav to show
```

Comment:
Just as an observation / question / other issue: There might be enough room on larger devices (tablets, landscape phones)?

Reply:
There might be, but:

a. It's difficult to reliably tell.
b. The UX is confusing if a single row contains tab icons and action icons.

I'll take a screenshot of that so you can see what I mean.

Reply:
Both of these are **without** the `onPrepareMenu` code in this PR.

Portrait on a phone:

![image](https://github.com/tuskyapp/Tusky/assets/773100/fe99b876-66cd-44a1-a5c0-a71c452bfd6d)

Portrait on a tablet:

![image](https://github.com/tuskyapp/Tusky/assets/773100/a380c96f-1eb3-45aa-9226-d5e912f50ba8)

If you pay careful attention to the colour and alignment of the search icon it's possible to tell that it's different, but not really at first glance.

Reply:
More or less what I would have expected from the tablet view: quite unfortunate (the spacing of the icons).

(But still a different issue.)